### ISSUE TITLE
X made no changes event in timeline

### ISSUE THREAD

**Issue by Bubu** [vector-im/riotX-android#1430]
I sometimes see `X made no changes` in the timeline in riotX. Riot-desktop filters such empty state events, I guess these shouldn't be shown in the timeline except when "Show hidden events" is selected from developer settings.

This is the event in question.
![image](https://user-images.githubusercontent.com/105185/83339240-13f3e300-a2cc-11ea-9e28-24d7ee151b31.png)

**Comment by bmarty** [comment]
Yes, I've noticed that too, and one question I have is: what the point of sending such (useless?) events?
@Half-Shot,  do you have an idea?

**Comment by Half-Shot** [comment]
The homeserver should be no-opping a join request here. The bridge will  join all rooms on startup to ensure it's joined, but it shouldn't be sending fresh join events into the room as the homeserver should be aware that the user is already in the room. Feels like a Synapse bug.

**Comment by Bubu** [comment]
Those events are still shown in the message preview in the chat list. I guess this is probably already tracked in better message previews in general somewhere?

**Comment by bmarty** [comment]
We will have to check again. I reopen the issue

**Comment by starcraft66** [comment]
I see this happen regularly.

**Comment by HarHarLinks** [comment]
Still seeing this today in v1.1.9

**Comment by piotrvidal** [comment]
I also see this issue happening regularly. Android version 12.

![Screenshot_20230808-134437-528.png](https://github.com/vector-im/element-android/assets/98980760/9cfffaf7-8c5f-4d93-9eea-15f098fbcd6c)

**Comment by erebion** [comment]
The issue still persists.

This also is inconsistent with Element Web/Desktop: https://github.com/element-hq/element-web/issues/23274

In Element Web/Desktop, this is only shown if "Show Hidden Events in Timeline" is enabled.

Those events should not be shown by default, as those are annoying to other people in a room.

**Comment by erebion** [comment]
> We will have to check again. I reopen the issue

@bmarty What did you find out while checking that? I'm hoping that's been something that'd help us here. :)

**Comment by SpiritCroc** [comment]
> Perhaps [@SpiritCroc](https://github.com/SpiritCroc) has an idea what might have fixed that in their fork. Then Element could just adopt the fix. :)

I do not remember touching that, but this issue is quite old so maybe I did, no idea 🤷 
Sounds like a straightforward change anyway
